### DISCUSSION TITLE
Install Third Party Integration

### DISCUSSION THREAD

**Discussion by brazorfajeje** [Security-Onion-Solutions/securityonion#13916] [Unsupported Versions]
Is it possible to install integrations not listed in https://docs.securityonion.net/en/2.4/third-party-integrations.html#supported-integrations but available for elastic https://www.elastic.co/docs/current/integrations ?

**Comment by InfosecGoon** [comment]
What particular integration are you interested in?

**Reply by brazorfajeje** [reply]
In my case was Cybereason integration https://www.elastic.co/guide/en/integrations/current/cybereason.html
but in general I would be interested in understanding how to handle the possibility of other integrations not supported directly by the development team
